doc/spire_server.md

```diff
@@ -68,6 +68,7 @@ This may be useful for templating configuration files, for example across differ
 | `ratelimit`                 | Rate limiting configurations, usually used when the server is behind a load balancer (see below)  |                                                                |
 | `socket_path`               | Path to bind the SPIRE Server API socket to                                                       | /tmp/spire-server/private/api.sock                             |
 | `trust_domain`              | The trust domain that this server belongs to (should be no more than 255 characters)              |                                                                |
+| `policy_engine`             | The policy engine used to determine authorization decision                                        | default SPIRE authorization policy                             |
```

Comment:
Agreed on the comments here! Will make these changes... only 1 question on one comment..

> server_full.conf needs to be updated with the new configs.

I think @azdagron had another comment from before on this that we may want to default to not setting it - thus using the default policy? This was to prevent replication of the policy in multiple files. Thoughts on the direction with this?

Reply:
current updated this to look like:

```
auth_policy_engine {
    opa_file_provider {
        ...
    }
}
```

Reply:
>I think @azdagron had another comment from before on this that we may want to default to not setting it - thus using the default policy? This was to prevent replication of the policy in multiple files. Thoughts on the direction with this?

@lumjjb `server_full.conf` is for documentation purposes only. It has all the possible config settings with comments so users can look at the different configurables in SPIRE. It doesn't look like the comment from @azdagron applies to that, right?

Reply:
ah i see - yea i got mixed up. Yea that works.